src/buzzer.h

```diff
@@ -4,14 +4,14 @@
 #include <stdint.h>
 
 typedef enum eBuzzFreq {
-	BUZZ_NONE=0,
-	BUZZ_2KHZ=3,
-	BUZZ_1KHZ=4,
-	BUZZ_500HZ=5,
-	BUZZ_250HZ=6
+	BUZZ_NONE = 0,
+	BUZZ_2KHZ = 3,
+	BUZZ_1KHZ = 4,
+	BUZZ_500HZ = 5,
+	BUZZ_250HZ = 6
```

Comment:
Legacy reasons because of the code origin where the value directly feeds a divider circuit. Not needed here.